react/features/conference/middleware.js

```diff
@@ -35,6 +39,12 @@ MiddlewareRegistry.register(store => next => action => {
 
         break;
     }
+    case KICKED_OUT: {
+        store.dispatch(
+            conferenceFailed(action.conference, JitsiConferenceEvents.KICKED));
+        store.dispatch(appNavigate(undefined));
```

Comment:
Maybe I was too quick quick with the approve button :-) While looking at this code I realized we currently have 3 types of conference errors:

- fatal errors: such as no bridge available, in this case we show the overlay
- recoverable errors: such as password required, in this case we "swallow" them and deal with them
- others: such as this one now, which we don't handle at all

In the last case, what happens is that the user remains in the conference view , but a spinning wheel will show up, though nothing is going to happen. There are other cases in which this can happen, AFAIS here: https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConferenceErrors.js

I think we should handle this situation. A quick proposal would be to handle CONFERENCE_FAILED and check if the failure is recoverable or fatal, if it's neither, navigate to undefined.

This can be done in a follow-up PR, of course.

Reply:
@saghul, thank you! I'll go on the escape route of "a follow-up PR."